stackslib/src/chainstate/stacks/boot/pox_4_tests.rs

```diff
@@ -1805,11 +2322,20 @@ fn delegate_stack_stx_extend_signer_key() {
     )
     .expect("No stacking state, stack-stx failed")
     .expect_tuple();
+    let delegation_state = get_delegation_state_pox_4(&mut peer, &latest_block, &alice_principal)
+        .expect("No delegation state, delegate-stx failed")
+        .expect_tuple();
 
-    let next_reward_cycle = 1 + burnchain
+    let stacking_state = get_stacking_state_pox_4(&mut peer, &latest_block, &alice_principal)
+        .expect("No stacking state, stack-stx failed")
```

Comment:
bob called delegate-stack-stx that failed here.

Should be changed in line 2323 as well.

Reply:
Can you explain more? When I run this test every contract call returns `ok`, including `delegate-stack-stx`